pep-0012/pep-NNNN.rst

```diff
@@ -6,6 +6,7 @@ PEP-Delegate: <PEP delegate's real name>
 Discussions-To: <email address or URL>
 Status: <REQUIRED: Draft | Active | Accepted | Provisional | Deferred | Rejected | Withdrawn | Final | Superseded>
 Type: <REQUIRED: Standards Track | Informational | Process>
+Content-Type: text/x-rst
```

Comment:
Content Type is required, but as there's only one valid value it feels odd to use the `<REQUIRED:  spam>` verbiage. Maybe it is better as is?

```suggestion
Content-Type: <REQUIRED: text/x-rst>
```

Reply:
Why is `Content-Type` still required?

Reply:
I had the same thought. One possible reason is forward compatibility: maybe in the future we'll allow PEPs in markdown or something. Then again, even in that case we can just reintroduce the header later.

And another reason is just to avoid making a change without a strong reason. There is little cost to having PEP authors copy this field from the template.

Reply:
I'm inclined to keep it, for clarity. Lots of things have their MIME types designated in headers. And as Jelle says, maybe in the future there will be others, so it'd mean we don't need to say "Optional - if omitted, defaults to text/x-rst".

Reply:
There's no techincal reason the header couldn't simply be removed from the template (and even all PEPs), since they all use `text/x-rst`, the default, and likely will for the immediate future, and the only place this was actually used is by the old docutils-based rendering script to determine the rendering mode, if not the default (it isn't anywhere in the output, and it isn't needed or used at all by the PEP 676 based system).

If we did add support for a new format, it presumably would be handled via file extension, like any common format these days (e.g. MyST with `.md`) and a `Content-Type` would not really serve no technical purpose, since it is only  (and could even be out of sync with the actual file type, unless we run a linter or custom build-time check to validate it). That would allow us to not have to mention it at all (optional or otherwise), lint it, or copy it as boilerplate.

That being said, while it is redundant, there isn't too much cost to keeping it around (at least on existing PEPs), so I don't feel too strongly about it.